src/screens/FeedScreen/RenderList.js

```diff
@@ -116,13 +116,12 @@ const RenderListFeed = (props) => {
 
   const isBlurred = item?.isBlurredPost && item?.anonimity;
 
-  const commentHeight = () => {
+  const addCommentPreviewHeight = () => {
+    const commentSectionHeight = getHeightReaction() - getHeightFooter();
     if (isBlurred && !hasComment) {
       return 0;
     }
-    const isSingleComment = getTotalReaction(item) === 1;
-    const commentSectionHeight = getHeightReaction() - getHeightFooter();
-    return isSingleComment ? commentSectionHeight - 20 : commentSectionHeight;
+    return commentSectionHeight;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function `addCommentPreviewHeight` has been simplified and the condition for a single comment has been removed. This might affect the height calculation for single comments. Please ensure that this change doesn't break the UI for single comments.
```diff
-    const isSingleComment = getTotalReaction(item) === 1;
-    return isSingleComment ? commentSectionHeight - 20 : commentSectionHeight;
+    return commentSectionHeight;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->